### ISSUE TITLE
Problem with `socket.io-prometheus-metrics`

### ISSUE THREAD

**Issue by pouriamoosavi** [jitsi/excalidraw-backend#13]
Hello hope you are well.

I was trying to install this module on a self hosted Jitsi server and I faced a problem with 9090 port. I had another program that was running on this port and I didn't want to stop it. 
I deleted this part and everything seems to be working correctly:
```
// listens on host:9090/metrics
prometheus.metrics(io, {
    collectDefaultMetrics: true
});
```

The thing is that I think we can make this (whatever it is) optional and enable it only if user asks for it.

As I had provided my desired port in .env file it was very confusing at first that it was asking for 9090 port to be open.

**Comment by saghul** [comment]
Good point. Would you like to send a PR making it optional with an env variable?

**Comment by alkanpolat** [comment]
Hi, can you help with this issue?

I set the port to 9091 in the index.ts file, but the drawings I made do not appear on the other user's screen. I get "WebSocket connection failed error" in the console

```
// listens on host:9090/metrics
prometheus.metrics(io, {
    port:9091,
    collectDefaultMetrics: true
});
```

**Comment by saghul** [comment]
Done in 2025.8.2
